README.md

```diff
@@ -158,6 +158,8 @@ Options:
   Security:
     --cosign
         Enable signing images with cosign.
+    --no-verify
```

Comment:
Even though it gets a bit long, I'd namespace it here to make it clear it's cosign verification which is disabled:

```suggestion
    --no-cosign-verify
```